Source/Core/InputCommon/ControlReference/FunctionExpression.cpp

```diff
@@ -265,7 +263,7 @@ class SmoothExpression : public FunctionExpression
   std::string GetFuncName() const override { return "smooth"; }
 
 private:
-  using Clock = std::chrono::steady_clock;
+  
```

Comment:
Could probably drop one (or two) of those empty lines here ;)

Reply:
Oops.